Tools/src/FormatCheckTool/FormatCheckTool.cs

```diff
@@ -0,0 +1,234 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Microsoft.Recognizers.Text;
+using Microsoft.Recognizers.Text.Choice;
+using Microsoft.Recognizers.Text.DateTime;
+using Microsoft.Recognizers.Text.Number;
+using Microsoft.Recognizers.Text.NumberWithUnit;
+using Microsoft.Recognizers.Text.Sequence;
+using Newtonsoft.Json;
+using Microsoft.Recognizers.Text.DataDrivenTests;
+
+namespace FormatCheckTool
```

Comment:
It's better to use a name space matching similar to that of the packages as this may in the future become its own DLL.
Maybe Microsoft.Recognizers.Text.Validation.